readme.txt

```diff
@@ -111,30 +111,42 @@ If you get stuck, you can ask for help in the [Plugin Forum](https://wordpress.o
 == Changelog ==
 
 = 9.7.0 - xxxx-xx-xx =
-* Update - Removes BNPL payment methods (Klarna and Affirm) when other official plugins are active
+
+**New Features**
+
+* Adds two new safety filters to the subscriptions detached debug tool: `wc_stripe_detached_subscriptions_maximum_time` and `wc_stripe_detached_subscriptions_maximum_count`
+* Show a notice when editing an active subscription that has no payment method attached
+* Show payment methods sync status on the UI
+* Add the number of pending webhooks to the Account status section
```

Comment:
I think we should show these features before the subscription tool/notices, as those are more specific to users who are also using WC Subscriptions.

Reply:
https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4509/commits/80d1e2d1791a6d804765bb14419534950c3b913f